eng/CodeAnalysis.src.globalconfig

```diff
@@ -489,6 +489,9 @@ dotnet_diagnostic.CA1868.severity = warning
 # CA1869: Cache and reuse 'JsonSerializerOptions' instances
 dotnet_diagnostic.CA1869.severity = warning
 
+# CA1870: Use a cached 'SearchValues' instance
```

Comment:
Can you add it to the test file as well (as none or suggestion) for consistency?